sample-apps/pathology/README.md

```diff
@@ -87,7 +93,31 @@ monailabel start_server --app workspace/pathology --studies datasets/wsi --conf
 # Start MONAI Label Server with HoVerNet Nuclei in Inference Only mode
 monailabel start_server --app workspace/pathology --studies datasets/wsi --conf models hovernet_nuclei --conf skip_trainers true
 ```
+### Usage (Development Mode)
+
+```bash
+git clone https://github.com/Project-MONAI/MONAILabel.git
+cd MONAILabel
+pip install -r requirements.txt
+```
+Install Openslide:
+
+> Install [Openslide](https://openslide.org/) binaries manually and make sure .dll or .so files for openslide are in system load path.
+> - For windows, make sure **&lt;openslide_folder&gt;**/bin is added in PATH environment.
+> - For ubuntu: `apt install openslide-tools`
+
+#### FileSystem as Datastore
 
+```bash
+# download sample wsi image (skip this if you already have some)
+mkdir sample_wsi
+cd sample_wsi
+wget https://demo.kitware.com/histomicstk/api/v1/item/5d5c07539114c049342b66fb/download
+cd -
+
+# run server
+./monailabel/scripts/monailabel start_server --app sample-apps/pathology --studies datasets/wsi
+```
```

Comment:
we also have DSA as datastore.. we have removed it somehow..

Reply:
Separated the PR and the part is merged. Thanks.